squad/frontend/templates/squad/_builds_table.jinja2

```diff
@@ -1,16 +1,24 @@
-<div class='highlight-row'>
+<div class='highlight-row' ng-app='Build' ng-controller='BuildComparisonController'>
+<input type="hidden" ng-init="compare_button_label='{{_('Compare builds')}}'" />
+<input type="hidden" ng-init="invalid_number_of_builds_msg='{{_('Two builds need to be selected!')}}'" />
+<input type="hidden" ng-init="invalid_selected_builds_msg='{{_('Selected builds must be different!')}}'" />
+<input type="button" ng-value="compare_button_label" class="btn btn-default" ng-click="compareBuilds('{{statuses.0.build.project.full_name}}')" />
```

Comment:
`<input>` elements require an enclosing `<form>` tag. it may work without it, but it's not valid HTML.